recipes/blas/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set variant = "openblas" %}
+
+package:
+  name: blas
+  version: 1.0.0
+
+build:
+  skip: true  # [win]
+  string: {{ variant }}
+
+requirements:
+  run:
+    - {{ variant }}
+
+test:
+  commands:
+    - conda list -n _test blas
+    - conda list -n _test --json -f blas | grep "blas-.*-{{ variant }}"
+
+about:
+  home: NA
```

Comment:
We were thinking of adding some document to explain how this works and then linking to it. Though we can certainly consider linking from there to the PR.